controller/Lists.php

```diff
@@ -436,6 +434,30 @@ public function removeListElement(): void
         $this->returnJson(['deleted' => $deleted]);
     }
 
+    private function sync(
```

Comment:
Maybe it would be clearer to name this as `loadRemoteList()` or `fetchRemoteList()` ?

I'd also try to avoid the boolean flag, maybe the error message can be built by the caller instead?